engine/build.gradle

```diff
@@ -103,9 +103,11 @@ dependencies {
     implementation group: 'net.logstash.logback', name: 'logstash-logback-encoder', version: '4.10'
 
     // Our developed libs
-    api group: 'org.terasology.gestalt', name: 'gestalt-asset-core', version: '7.1.0'
-    api group: 'org.terasology.gestalt', name: 'gestalt-module', version: '7.1.0'
-    api group: 'org.terasology.gestalt', name: 'gestalt-util', version: '7.1.0'
+    api group: 'org.terasology.gestalt', name: 'gestalt-asset-core', version: '7.1.0-SNAPSHOT'
+    api group: 'org.terasology.gestalt', name: 'gestalt-module', version: '7.1.0-SNAPSHOT'
+    api group: 'org.terasology.gestalt', name: 'gestalt-entity-system', version: '7.1.0-SNAPSHOT'
+    api group: 'org.terasology.gestalt', name: 'gestalt-util', version: '7.1.0-SNAPSHOT'
+    api "com.github.zafarkhaja:java-semver:0.10.0"  // ASAP: Remove after https://github.com/MovingBlocks/gestalt/pull/110
```

Comment:
https://github.com/MovingBlocks/gestalt/pull/110 has been merged - do we still need this?

Reply:
^ pinging @keturn @DarkWeird